components/apimgt/org.wso2.carbon.apimgt.throttling.service/src/gen/java/org/wso2/carbon/throttle/service/factories/RevokedjwtApiServiceFactory.java

```diff
@@ -0,0 +1,14 @@
+package org.wso2.carbon.throttle.service.factories;
+
+import org.wso2.carbon.throttle.service.RevokedjwtApiService;
+import org.wso2.carbon.throttle.service.impl.RevokedjwtApiServiceImpl;
+
+public class RevokedjwtApiServiceFactory {
+
+   private final static RevokedjwtApiService service = new RevokedjwtApiServiceImpl();
+
+   public static RevokedjwtApiService getRevokedjwtApi()
```

Comment:
```suggestion
   public static RevokedjwtApiService getRevokedjwtApi() {
```

Reply:
This a generated class. Will format anyways.